### DISCUSSION TITLE
Table definitions can display invalid SQL

### DISCUSSION THREAD

**Discussion by zqianem** [supabase/supabase#16944] [Contribute to Supabase]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

Table definitions (#13384) can display invalid SQL because column names that are SQL keywords are not quoted.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Visit the table editor for a project
2. Create a table with a column named "desc"
3. Switch the view to "Definition" (top-right toggle)
4. See invalid SQL

## Expected behavior

Table definitions are valid SQL; column names that are SQL keywords are quoted.

## Screenshots

![image](https://user-images.githubusercontent.com/47396035/233476693-a91f02c2-32d6-461d-9bd2-cc5a1dcd7c8b.png)

**Comment by saltcod** [comment]
Another report via feedback: 

`If the definition is copied and pasted into the SQL Editor to create a table any camelCase columns are converted to lowercase`
